docs/usage.rst

```diff
@@ -1,4 +1,74 @@
 Usage
 =====
 
-TODO: Explain how to use your library.
+Prerequisites
+-------------
+
+To perform its function, django-critical-css requires:
+
+* `penthouse <https://www.npmjs.com/package/penthouse/>`_ service available.
+* redis worker available.
+
+
+Templatetag for inlining critical css
+-------------------------------------
+
+django-critical-css provides the `critical_css` templatetag to inline critical css
+from the database into your template using ``templates/critical/critical.html``.
+
+.. code-block:: text
+
+    {% load critical_tags %}
+
+      {% critical_css 'your/css/styles_path.css' %}
```

Comment:
Bring `{% load critical_tags %}` and `{% critical_css ... %}` to same indentation:

```suggestion
    {% critical_css 'your/css/styles_path.css' %}
```